dynamixel_controllers/src/dynamixel_controllers/joint_trajectory_action_controller.py

```diff
@@ -77,35 +76,50 @@ def __init__(self, controller_namespace, controllers):
 
         self.port_to_joints = {}
         for c in controllers:
-            if c.port_namespace not in self.port_to_joints: self.port_to_joints[c.port_namespace] = []
+            if c.port_namespace not in self.port_to_joints:
+                self.port_to_joints[c.port_namespace] = []
             self.port_to_joints[c.port_namespace].append(c.joint_name)
 
         self.port_to_io = {}
         for c in controllers:
-            if c.port_namespace in self.port_to_io: continue
+            if c.port_namespace in self.port_to_io:
+                continue
             self.port_to_io[c.port_namespace] = c.dxl_io
 
         # added motor bias [AHC: 20171127]
         self.joint_to_bias = {}
         for c in controllers:
             self.joint_to_bias[c.joint_name] = c.motor_bias
 
-        self.joint_states = dict(list(zip(self.joint_names, [c.joint_state for c in controllers])))
+        self.joint_states = dict(
+            list(zip(self.joint_names, [c.joint_state for c in controllers]))
+        )
         self.num_joints = len(self.joint_names)
-        self.joint_to_idx = dict(list(zip(self.joint_names, list(range(self.num_joints)))))
-
-    def initialize(self):        self.joint_to_controller = {}
```

Comment:
This formatting was very broken in python3, alongside some bad indentation issues elsewhere.